Cargo.toml

```diff
@@ -45,12 +45,11 @@ indoc = "1.0.7"
 [target.'cfg(windows)'.dependencies]
 windows = { version = "0.44.0", features = ["Win32_Foundation", "Win32_UI_Shell", "Win32_System_Console", "Services_Store", "Foundation", "Foundation_Collections", "Web_Http", "Storage_Streams",] }
 
-[target.'cfg(target_os = "macos")'.dependencies]
-ureq = { version = "2", features = ["native-tls"] }
-native-tls = "0.2.10"
+[target.'cfg(not(any(target_os = "windows", target_env = "musl")))'.dependencies]
+reqwest = { version = "0.11.16",  default-features = false, features = ["blocking", "native-tls"] }
 
-[target.'cfg(all(not(target_os = "macos"),not(windows)))'.dependencies]
-ureq = { version = "2", features = ["native-certs"] }
+[target.'cfg(all(target_os = "linux", target_env = "musl"))'.dependencies]
+reqwest = { version = "0.11.16",  default-features = false, features = ["blocking", "native-tls-vendored"] }
```

Comment:
I think you want the entire block for `reqwest` look like this:
```
[target.'cfg(target_os = "macos")'.dependencies]
reqwest = { version = "0.11", default-features = false, features = ["blocking", "native-tls"] }

[target.'cfg(all(not(target_os = "macos"),not(windows)))'.dependencies]
reqwest = { version = "0.11", default-features = false, features = ["blocking", "rustls-tls-native-roots"] }
```

I think that would be equivalent to how we handle this right now in terms of using certificates etc. I think you then also don't need to worry about installing `libssl` etc.

Reply:
I just pushed a commit that fixes this.